examples/persons/src/EditPerson.re

```diff
@@ -11,6 +11,34 @@ module EditPersonMutation = [%graphql
   |}
 ];
 
+module OptimisticResponse = {
+  type t = {
```

Comment:
can you add a comment that this is necessary because the types graphql_ppx_re generates are not 1:1 to the JS representation and probably this won't be necessary as soon as https://github.com/baransu/graphql_ppx_re/pull/20 gets merged?

Reply:
I didn't have time to check back in before now. Would you still like me to add it?

Reply:
would be appreciated! I haven't published yet

Reply:
On it.